admin/netdata/Makefile

```diff
@@ -18,7 +18,7 @@ PKG_CPE_ID:=cpe:/a:my-netdata:netdata
 
 PKG_SOURCE:=$(PKG_NAME)-v$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/netdata/netdata/tar.gz/v$(PKG_VERSION)?
-PKG_HASH:=94492108a6e24e8b39c011ae35ff6f50a848d816af396fdf2b44655cecd78672
+PKG_HASH:=032f9001e2b7f774989a200519feeafa3a4d9b7c9e8fb0cfa43e68a8da6ad1bf
 
 PKG_INSTALL:=1
```

Comment:
Out of curiosity, does PKG_BUILD_PARALLEL help here?

Reply:
Good question. AFAIK: It should, but to be sure, I'm going to test it multiple times during the weekend with full builds. (Reference: https://gitlab.labs.nic.cz/turris/turris-build/commit/7646aa98e86463ae0fd04e0ce5e1173685705d26 )

Reply:
I'm not sure if it's the toolchain logic itself or make but I'm seeing weird random breakage compared to other distributions which makes me a bit hesitant to add that flag.

Reply:
I'd add it with 0 to make it explicit.